To: vim_dev@googlegroups.com
Subject: Patch 7.3.175
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.175
Problem:    When 'colorcolumn' is set locally to a window, ":new" opens a
	    window with the same highlighting but 'colorcolumn' is empty.
	    (Tyru)
Solution:   Call check_colorcolumn() after clearing and copying options.
	    (Christian Brabandt)
Files:	    src/buffer.c


*** ../vim-7.3.174/src/buffer.c	2011-04-11 16:56:29.000000000 +0200
--- src/buffer.c	2011-05-05 17:28:24.000000000 +0200
***************
*** 2527,2532 ****
--- 2527,2535 ----
      if (p_fdls >= 0)
  	curwin->w_p_fdl = p_fdls;
  #endif
+ #ifdef FEAT_SYN_HL
+     check_colorcolumn(curwin);
+ #endif
  }
  
  /*
*** ../vim-7.3.174/src/version.c	2011-05-05 17:23:58.000000000 +0200
--- src/version.c	2011-05-05 17:30:32.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     175,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
55. You ask your doctor to implant a gig in your brain.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
